frontend/javascripts/admin/organization/upgrade_plan_modal.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { useRef } from "react";
+import { Divider, InputNumber, Modal } from "antd";
+import moment from "moment";
+import {
+  DatabaseOutlined,
+  FieldTimeOutlined,
+  RocketOutlined,
+  UserAddOutlined,
+} from "@ant-design/icons";
+import { APIOrganization } from "types/api_flow_types";
+import { formatDateInLocalTimeZone } from "components/formatted_date";
+import {
+  sendExtendPricingPlanEmail,
+  sendUpgradePricingPlanEmail,
+  sendUpgradePricingPlanStorageEmail,
+  sendUpgradePricingPlanUserEmail,
+} from "admin/admin_rest_api";
+import { powerPlanFeatures, teamPlanFeatures } from "./pricing_plan_utils";
+import { PricingPlanEnum } from "./organization_edit_view";
+
+function extendPricingPlan(organization: APIOrganization) {
+  const extendedDate = moment(organization.paidUntil).add(1, "year");
+
+  Modal.confirm({
+    title: "Extend Current Plan",
+    okText: "Request Extension",
+    onOk: sendExtendPricingPlanEmail,
+    icon: <FieldTimeOutlined style={{ color: "var(--ant-primary-color)" }} />,
+    width: 1000,
+    content: (
+      <div
+        style={{
+          background:
+            "right -150px / 50% no-repeat url(/assets/images/pricing/background_neurons.png)",
+        }}
+      >
+        <p style={{ marginRight: "30%" }}>
+          Extend your plan now for uninterrupted access to webKnossos. Expired plans will be
+          downgrade to the Free plan and you might lose access to some webKnossos features and see
+          restrictions on the number of permitted user accounts and your included storage space
+          quota.
+        </p>
+        <p>
+          Your current plan is paid until:{" "}
+          {formatDateInLocalTimeZone(organization.paidUntil, "YYYY-MM-DD")}
+        </p>
+        <p>Buy extension until: {extendedDate.format("YYYY-MM-DD")}</p>
```

Comment:
@fm3 @frcroth What is the default `paidUntil` value for the `Free` plan? Also `null`?

@philippotto Do you have an expiration date in your DB? In case, the default value is `null` then I need to handle that as well.

Reply:
Yes I think null/None makes sense for paidUntil in Free. The Free plan should not semantically have an “expiration date”, right?

Reply:
> @philippotto Do you have an expiration date in your DB? In case, the default value is null then I need to handle that as well.

No, I hadn't added an expiration date to the DB when testing. So, handling null is probably the right way to fix this.

Reply:
`paidUntil == null` => no expiration

Reply:
Fixed